index.html

```diff
@@ -43,6 +43,263 @@
+                            <li><a href="#">Ruby</a></li>
+                        </ul>
+                    </div> 
+                </div>
+                <div class="button_bg">
+                    <div class="btn">
+                        <a href="#"> See Projects</a>
+                    </div>
+                </div>
+            </div>
+            <div class="posts">
+                <div class="p_title">
+                    <h1>Profesional Art <br>Printing Data</h1>
+                    <p>
+                        A daily selection of privately 
+                        <br>personalized reads; no accounts or
+                        <br>sign-ups required. Has been the 
+                        <br>industry's standard.
+                    </p>
+                    <div class="p_p">
+                        <ul>
+                            <li><a href="#">HTML</a></li>
+                            <li><a href="#">Bootstrap</a></li>
+                            <li><a href="#">Ruby</a></li>
+                        </ul>
+                    </div> 
+                </div>
+                <div class="button_bg">
+                    <div class="btn">
+                        <a href="#"> See Projects</a>
+                    </div>
+                </div>
+            </div>
+            <div class="posts">
+                <div class="p_title">
+                    <h1>Profesional Art <br>Printing Data</h1>
+                    <p>
+                        A daily selection of privately 
+                        <br>personalized reads; no accounts or
+                        <br>sign-ups required. Has been the 
+                        <br>industry's standard.
+                    </p>
+                    <div class="p_p">
+                        <ul>
+                            <li><a href="#">HTML</a></li>
+                            <li><a href="#">Bootstrap</a></li>
+                            <li><a href="#">Ruby</a></li>
+                        </ul>
+                    </div> 
+                </div>
+                <div class="button_bg">
+                    <div class="btn">
+                        <a href="#"> See Projects</a>
+                    </div>
+                </div>
+            </div>
+            <div class="posts">
+                <div class="p_title">
+                    <h1>Profesional Art <br>Printing Data</h1>
+                    <p>
+                        A daily selection of privately 
+                        <br>personalized reads; no accounts or
+                        <br>sign-ups required. Has been the 
+                        <br>industry's standard.
+                    </p>
+                    <div class="p_p">
+                        <ul>
+                            <li><a href="#">HTML</a></li>
+                            <li><a href="#">Bootstrap</a></li>
+                            <li><a href="#">Ruby</a></li>
```

Comment:
Great work on implementing this section, but kindly I suggest you remove the link (`a` tag) in the stacks used for each project. 😄